util/output.js

```diff
@@ -0,0 +1,24 @@
+var exports = module.exports = {};
+
+var readline = require('readline');
+
+exports.lineOutput = function (socket, process, event, increment) {
+  var donePercent = 0;
+  readline.createInterface({
+    input     : process.stdout,
+    terminal  : false
```

Comment:
No particular reason. I found this solution at https://stackoverflow.com/a/20272799/4592183